plugins/janus_audiobridge.c

```diff
@@ -2219,11 +2263,43 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 		pkt->seq_number = ntohs(rtp->seq_number);
 		/* Any audio level extension to quickly check if this is silence? */
 		pkt->silence = FALSE;
+
 		if(participant->extmap_id > 0) {
 			int level = 0;
-			if(janus_rtp_header_extension_parse_audio_level(buf, len, participant->extmap_id, &level) == 0) {
+			if (janus_rtp_header_extension_parse_audio_level(buf, len, participant->extmap_id, &level) == 0) {
+				participant->audio_dBov_sum = participant->audio_dBov_sum + level;
+				participant->audio_active_packets = participant->audio_active_packets + 1;
 				pkt->silence = (level == 127);
 				participant->dBov_level = level;
+				if (participant->room->audiolevel_event) {
+					if (participant->audio_active_packets == participant->room->audio_active_packets) {
+
+						if ((float) participant->audio_dBov_sum / (float) participant->audio_active_packets <
```

Comment:
Shouldn't this check if `participant->audio_active_packets` is different than zero? Otherwise that math operation might result in a divide by zero and crash Janus.